browse/templates/abs/extra_services.html

```diff
@@ -42,10 +42,10 @@
   </div>
 
   {%- set current = abs_meta.version == abs_meta.version_history[-1].version -%}
-  {%- if format_list|length == 0 or ( not current and format_list|length == 1 and format_list[0] == 'src' ) -%}
-  <li>Unavailable</li>
-  {%- elif withdrawn -%}
+  {%- if withdrawn -%}
```

Comment:
withdrawn check needs to go before format check. Withdrawn versions sometimes have a small non-empty source file.